subworkflows/local/benchmarking.nf

```diff
@@ -0,0 +1,114 @@
+include { GNU_SORT as SORT          } from '../../modules/nf-core/gnu/sort'
+include { BEDTOOLS_MERGE            } from '../../modules/nf-core/bedtools/merge'
+include { BEDTOOLS_INTERSECT        } from '../../modules/nf-core/bedtools/intersect'
+include { BEDTOOLS_JACCARD          } from '../../modules/nf-core/bedtools/jaccard'
+include { BEDTOOLS_GENOMECOV        } from '../../modules/nf-core/bedtools/genomecov'
+include { BENCHMARKING_MULTIQC as JACCARD_MULTIQC } from '../../modules/local/benchmarking/multiqc'
+include { BENCHMARKING_MULTIQC as CORRELATION_MULTIQC } from '../../modules/local/benchmarking/multiqc'
+include { PNG_JSON as LOCATION_JSON } from '../../modules/local/benchmarking/png_json'
+include { PNG_JSON as OVERLAP_JSON  } from '../../modules/local/benchmarking/png_json'
+include { LOCATION_PLOT             } from '../../modules/local/benchmarking/location_plots'
+include { OVERLAP_PLOT              } from '../../modules/local/benchmarking/overlap_plot'
+include { SEQ_DEPTH_CORRELLATION    } from '../../modules/local/benchmarking/seq_depth_plot'
+include { AVERGAGE_TSV              } from '../../modules/local/benchmarking/average_tsv'
+
+
+
+workflow BENCHMARKING {
+
+    take:
+    ch_real_bed
+    ch_benchmarking_bed
+    ch_real_bam
+    ch_benchmarking_bam
+    ch_trim_report
+
+    main:
+
+    //data preparation
+    ch_versions = Channel.empty()
+
+    ch_all = ch_real_bed.mix(ch_benchmarking_bed)
+        .map{ meta, bed -> [[id: meta.tool + "_" + (meta.benchmarking ? "benchmarking" : "real"),
+                            tool: meta.tool,
+                            benchmarking: meta.benchmarking], bed]}
+        .groupTuple()
+
+
+    SORT(ch_all)
+
+    BEDTOOLS_MERGE(SORT.out.sorted).bed.branch{ meta, bed ->
+            real: !meta.benchmarking
+            benchmarking: meta.benchmarking
+        }.set { ch_merged }
+
+    ch_joined = ch_merged.real.map{ meta, bed -> [[id: meta.tool], bed]}
+        .join(ch_merged.benchmarking.map{ meta, bed -> [[id: meta.tool], bed]})
+
+    //Overlap plot
+    ch_intersect = BEDTOOLS_INTERSECT(ch_joined,[[], []])
+    OVERLAP_PLOT(ch_intersect)
+    OVERLAP_JSON(OVERLAP_PLOT.out, "Overlap plots", "Plot the overlap circRNAs found in total and polyA data for the tools")
```

Comment:
Ok I now see the point of the `PNG_JSON`, but please change the input definition to a single tuple and call it like this:
`OVERLAP_JSON(OVERLAP_PLOT.out.map{plot -> [plot, 'Title', 'Description']})`

This way, it is also possible to insert multi-value channels